vedro_jira_failed_reporter/_failed_reporter_plugin.py

```diff
@@ -51,8 +51,8 @@ def subscribe(self, dispatcher: Dispatcher) -> None:
         if self._report_enabled:
             dispatcher.listen(ScenarioFailedEvent, self.on_scenario_failed)
 
-    def _make_search_test_file_link(self, traceback: TracebackType) -> str:
-        return get_traceback_entrypoint_filename(traceback)
+    def _make_search_test_file_link(self, scenario_result: ScenarioResult) -> str:
```

Comment:
Unnecessary nesting then.

let's get rid of _make_search_test_file_link:
```
test_file = str(scenario_result.scenario.rel_path)
```
from `_make_new_issue_description_for_test` and from `on_scenario_failed`

Reply:
Done